modules/adding-ibmz-lpar-agent.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="adding-ibmz-lpar-agents_{context}"]
+= Adding {ibm-z-title} agents in a Logical Partition (LPAR)
+
+Use the following procedure to manually add {ibm-z-name} agents to your cluster that runs in an LPAR environment. Use this procedure only for {ibm-z-name} clusters running in an LPAR.
+
+.Prerequisites
+* You have Python 3 installed.
+.Procedure
+
+. Create a boot parameter file for the agents.
++
+.Example parameter file
+[source,terminal]
+----
+rd.neednet=1 cio_ignore=all,!condev \
+console=ttysclp0 \
+ignition.firstboot ignition.platform.id=metal
+coreos.live.rootfs_url=http://<http_server>/rhcos-<version>-live-rootfs.<architecture>.img \// <1>
+coreos.inst.persistent-kargs=console=ttysclp0
+ip=<ip>::<gateway>:<netmask>:<hostname>::none nameserver=<dns> \// <2>
+rd.znet=qeth,<network_adaptor_range>,layer2=1
+rd.<disk_type>=<adapter> \// <3>
+zfcp.allow_lun_scan=0
+ai.ip_cfg_override=1 \// <4>
+random.trust_cpu=on rd.luks.options=discard
+----
+<1> For the `coreos.live.rootfs_url` artifact, specify the matching `rootfs` artifact for the `kernel` and `initramfs` that you are starting. Only HTTP and HTTPS protocols are supported.
+<2> For the `ip` parameter, manually assign the IP address, as described in _Installing a cluster with z/VM on IBM Z and IBM LinuxONE_.
+<3> For installations on DASD-type disks, use `rd.dasd` to specify the DASD where RedHat Enterprise Linux CoreOS (RHCOS) is to be installed. For installations on FCP-type disks, use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where RHCOS is to be installed.
+<4> Specify this parameter when you use an Open Systems Adapter (OSA) or HiperSockets.
+
+. Generate the `.ins` and `initrd.img.addrsize` files.
++
+The `.ins` file is a special file that includes installation data and is present on the FTP server. It can be accessed from the HMC system.
+This file contains details such as mapping of the location of installation data on the disk or FTP server, the memory locations where the data is to be copied.
++
+[NOTE]
+====
+The `.ins` and `initrd.img.addrsize` files are not automatically generated as part of boot-artifacts from the installer. You must manually generate these files.
+====
+
+.. Save the following script to a file, such as `generate-files.py`:
++
+.`generate-files.py` file
+[source,python]
+----
+# The following commands retrieve the size of the `kernel` and `initrd`:
```

Comment:
I don't think the dollar sign means much in Python code when it comes to running Python commands. Worth checking with the SME. With functions, methods, and so on that are called rather than run, the dollar sign would only confuse the naming convention for Python instances.